src/OneMightyRoar/PHP_Paulus_Components/BasicApp.php

```diff
@@ -155,10 +155,10 @@ public function getEnvironment()
      * @access public
      * @return BasicApp
      */
-    public function prepare($auto_load_routes = null, RouteLoader $route_loader = null)
+    public function prepare($auto_load_routes = true, RouteLoader $route_loader = null)
     {
         // Change our defaults
-        $auto_load_routes = (null === $auto_load_routes) ? true : false;
+        $auto_load_routes = (null === $auto_load_routes) ? true : $auto_load_routes;
```

Comment:
I'm not losing sleep over it, I just wouldn't have this set a precedent where we need to null check every scalar param with a default.